.stylelintrc

```diff
@@ -0,0 +1,21 @@
+{
+    "plugins": [
+        "stylelint-scss"
+    ],
+    "extends": [
+        "stylelint-config-standard"
+    ],
+    "rules": {
+        "at-rule-no-unknown": null,
```

Comment:
We use some SASS-specific directives that are flagged as unrecognized by stylelint out of the box - they made the deliberate decision to only include pure CSS @ rules.  So this rule is turned off here, and instead we use the stylelint-scss plugin and enable the `scss/at-rule-no-unknown' rule it provides below to offer the same coverage, just with built-in awareness of standard sass functions, etc.